python/cudf/cudf/_lib/groupby.pyx

```diff
@@ -174,18 +166,100 @@ cdef class GroupBy:
 
         try:
             with nogil:
-                if scan:
-                    c_result = move(
-                        self.c_obj.get()[0].scan(
-                            c_agg_requests
-                        )
+                c_result = move(
+                    self.c_obj.get()[0].aggregate(
+                        c_agg_requests
+                    )
+                )
+        except RuntimeError as e:
+            # TODO: remove this try..except after
+            # https://github.com/rapidsai/cudf/issues/7611
+            # is resolved
+            if ("make_empty_column") in str(e):
+                raise NotImplementedError(
+                    "Aggregation not supported for empty columns"
+                ) from e
+            else:
+                raise
```

Comment:
Can we now remove this todo since https://github.com/rapidsai/cudf/issues/7611 is resolved?